conseil-api/src/main/resources/application.conf

```diff
@@ -28,7 +40,10 @@ conseil {
   security.apiKeys {
     # List of authorized API keys needed to query Conseil
     keys: []
+    keys+= ${?CONSEIL_API_SECURITY_KEY}
+    
     allow-blank: false
+    allow-blank: ${?CONSEIL_API_SECURITY_ALLOW_BLANK}
```

Comment:
Please consider CONSEIL_API_ALLOW_BLANK_KEYS

Reply:
Sounds better. Wonder if we should put `SECURITY` in between.

Reply:
see my previous comment regarding my reservations